### DISCUSSION TITLE
MkDocs plugin to select Pygments styles for light and dark color schemes

### DISCUSSION THREAD

**Discussion by pawamoy** [pygments/pygments#2711] [Show and tell]
I just released a project (for my sponsors only) called MkDocs Pygments: https://pawamoy.github.io/mkdocs-pygments/.
I'm reaching out here because of these two paragraphs in my docs:

> The left column respects the theme's background color. The right column ignores it and uses the background color of the current color scheme (default or slate). You will notice that some themes work well with both schemes, while others do not. To fix a specific theme, we invite users to create light or dark variants for this theme and send pull requests to [Pygments](https://github.com/pygments/pygments?tab=readme-ov-file) (please read their [contributing instructions](https://pygments.org/docs/contributing/)).

> You will also notice that for some themes, names, punctuation, operators or other kinds of tokens are very hard to see depending on the current color scheme (default or slate): it's because these themes do not define colors for names, punctuation, operators, etc., and these unstyled tokens then take the color set by the current color scheme and the selected theme of our own docs pages. To fix a specific theme, we invite users to open issues on [Pygments](https://github.com/pygments/pygments?tab=readme-ov-file) (please read their [contributing instructions](https://pygments.org/docs/contributing/)) to ask whether this theme could be improved by styling additional tokens, and to send pull requests if maintainers accept them.

Let me know if anything is incorrect, or if you would prefer that I do not direct users to the Pygments repository (I don't have a lot of users, don't worry).

And thank you for Pygments :rocket:!

**Comment by Anteru** [comment]
Hey, congrats on the release and integrating Pygments with more tools. Our GitHub community is now at 722,299 dependent repositories, so a few users more won't move the needle significantly :) Happy if you redirect them to our page, we're always happy to take contributions.

**Reply by pawamoy** [reply]
Thanks!

Wow that's a lot of dependent projects indeed :sweat_smile: Congratulations to you too!

**Reply by pawamoy** [reply]
Oh by the way, does Pygments accept financial contributions? I'd be happy to give back a bit :slightly_smiling_face:
